lib/common.sh

```diff
@@ -134,6 +134,26 @@ is_ip_port_reachable()
     nc -w 3 -z $ip $port > /dev/null 2>&1
 }
 
+#
+# Check if the desired entry is present in /etc/hosts
+#
+is_present_in_etc_hosts() {
+    local ip="$1"
+    local hostname="$2"
+
+    while IFS= read -r entry; do
+        # Check if the line is not commented
+        if ! [[ "$entry" =~ ^[[:space:]]*# ]]; then
+            # Check if corresponding entry for IP and hostname is present
+            if echo "$entry" | grep -qE "^[[:space:]]*$ip[[:space:]]*$hostname"; then
```

Comment:
@linuxsmiths should the regex be "^[[:space:]]*$ip[[:space:]]*$hostname**$**" to avoid anything else after this host name?

Reply:
A valid format for /etc/hosts could be "_ip-address FQDN Placeholder-for-FQDN_"
@linuxsmiths we have to handle this case as well right?